docs/content/querying/dimensionspecs.md

```diff
@@ -194,11 +200,11 @@ Example for the `__time` dimension:
 }
 ```
 
-### Lookup extraction function
+### Lookup Extraction Function
 
 Lookups are a concept in Druid where dimension values are (optionally) replaced with new values. 
-For more documentation on using lookups, please see [here](../querying/lookups.html). 
-Explicit lookups allow you to specify a set of keys and values to use when performing the extraction.
+A lookup can be of type `namespace` or `map`. A `map` lookup is passed as part of the query. 
```

Comment:
All right. I'll close this PR and I'm looking forward to see the current developments released.